runtime/jcl/common/jclexception.c

```diff
@@ -372,3 +375,21 @@ getStackTrace(J9VMThread * vmThread, j9object_t * exceptionAddr, UDATA pruneCons
 
 	return result;
 }
+
+/**
+ * Check if a classloader is a built-in classloader.
+ *
+ * @param classLoader The classloader to check.
```

Comment:
The J9JavaVM parameter isn't documented. Also I'd switch the order and pass J9JavaVM first, I think this is the more common practice.